bin/tessel-2.js

```diff
@@ -19,9 +19,13 @@ const CLI_ENTRYPOINT = 'cli.entrypoint';
 
 // Check for updates
 const pkg = require('../package.json');
-updateNotifier({
-  pkg
-}).notify();
+
+try {
+  updateNotifier({ pkg }).notify();
+} catch (err) {
+  logs.warn('Failed to check for newer version of the CLI');
```

Comment:
I might even be in favor of silently failing with this one because it seems like people who do have the problem will hit this virtually every time they use the CLI.